src/gen/conditions.rs

```diff
@@ -2412,6 +2461,17 @@ fn test_duplicate_create_coin() {
     );
 }
 
+#[test]
+fn test_duplicate_create_coin_with_hint() {
```

Comment:
Do we support more than one extra argument as "hints" ?

Reply:
we do. This test is somewhat unrelated to the PR, but I noticed it was missing. This test is demonstrating that the hint field of `NewCoin` is not considered when looking for duplicate outputs. i.e. that we create two coins, where only one of them includes a hint.

Reply:
by "support" I mean: we allow them, but we only care about the first one (for now)

Reply:
Gotcha!